CMakeLists.txt

```
@@ -0,0 +1,124 @@
+#*****************************************************************************
+# DESCRIPTION: Script for build tool cmake on both unix and windows
+#
+# This file is part of Verilator.
+#
+# Code available from: https://verilator.org
+#
+#*****************************************************************************
+#
+# Copyright 2003-2022 by Wilson Snyder. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+#
+#****************************************************************************/
+
+cmake_minimum_required(VERSION 3.15)
+cmake_policy(SET CMP0091 NEW) # Use MSVC_RUNTIME_LIBRARY to select the runtime
+project(Verilator
+    VERSION 5.003
+    HOMEPAGE_URL https://verilator.org
+    LANGUAGES CXX
+)
+
+option(CMAKE_UNITY_BUILD
+    "Builds concatenated source files, potentially sppeding up the build.")
```

Is this file setup to support this option?

```suggestion
option(CMAKE_UNITY_BUILD
    "Builds concatenated source files, potentially speeding up the build.")
```

---

I haven't tested CMAKE_UNITY_BUILD

---

Remove this then?  Or test?

---

I removed it